source/includes/connection-snippets/x509-snippets/python-connection.py

```diff
@@ -0,0 +1,12 @@
+#
+begin x509 connection
```

Comment:
Did you mean for this to be on a separate line or should it be part of line 1?

Reply:
I did not mean this to be a separate line. Fixed, thanks for pointing it out